### ISSUE TITLE
Provide some simple discretizations of the unit box

### ISSUE THREAD

**Issue by cafaxo** [gaioguys/GAIO.jl#30]
The user shouldn't have to write this code: https://github.com/gaioguys/GAIO.jl/blob/88b26aca4e953460e8fff7afcc4b55691e59020d/examples/henon.jl#L4-L9

**Comment by gaioguy** [comment]
I agree. I guess we have two options: 

1. Provide some `SamplePoints` struct which takes two input parameters specifying the type of sample points and their overall number.
1. Provide a constructor for `PointDiscretizedMap` which takes these two arguments and internally constructs the points.

Any opinions on this?
... and, yes, that does not matter (currently).

**Comment by MaHermann** [comment]
Why not simply have some functions along the lines of 
```julia
dimension = 3
n = 10
points = corners(dimension,n)
points = randomPoints(dimension,n)
points = regularGrid(dimension,n)
```

These would just return an array of points and then we can give that to a `BoxMap` as before. Possibly we can also have methods of these that take a `Box` and use its dimension, might be more convenient.

**Comment by gaioguy** [comment]
Right: keep it simple.  I like that, but would use `facePoints` instead of `corners`.
